backend/internal/system/export/service.go

```diff
@@ -0,0 +1,285 @@
+		options = &ExportOptions{
+			Format: formatYAML,
+		}
+	}
+	if options.Format == "" {
+		options.Format = formatYAML
+	}
+
+	var exportFiles []ExportFile
+	var exportErrors []ExportError
+	resourceCounts := make(map[string]int)
+
+	// Export applications if requested
+	if len(request.Applications) > 0 {
+		appFiles, appErrors := es.exportApplications(request.Applications, options)
+		exportFiles = append(exportFiles, appFiles...)
+		exportErrors = append(exportErrors, appErrors...)
+		resourceCounts["applications"] = len(appFiles)
+	}
+
+	if len(exportFiles) == 0 {
+		return nil, serviceerror.CustomServiceError(
+			ErrorNoResourcesFound,
+			"No valid resources found for export",
+		)
+	}
+
+	// Calculate total size
+	var totalSize int64
+	for i := range exportFiles {
+		exportFiles[i].Size = int64(len(exportFiles[i].Content))
+		totalSize += exportFiles[i].Size
+	}
+
+	summary := &ExportSummary{
+		TotalFiles:    len(exportFiles),
+		TotalSize:     totalSize,
+		ExportedAt:    time.Now().UTC().Format(time.RFC3339),
+		ResourceTypes: resourceCounts,
+		Errors:        exportErrors,
+	}
+
+	return &ExportResponse{
+		Files:   exportFiles,
+		Summary: summary,
+	}, nil
+}
+
+// exportApplications exports application configurations as YAML files.
+func (es *exportService) exportApplications(applicationIDs []string, options *ExportOptions) (
+	[]ExportFile, []ExportError) {
+	logger := log.GetLogger().With(log.String("component", "ExportService"))
+	exportFiles := make([]ExportFile, 0, len(applicationIDs))
+	exportErrors := make([]ExportError, 0, len(applicationIDs))
+
+	applicationIDList := make([]string, 0)
+	if len(applicationIDs) == 1 && applicationIDs[0] == "*" {
+		// TODO: Implement GetAllApplications method or pagination
+		apps, err := es.applicationService.GetApplicationList()
+		if err != nil {
+			logger.Warn("Failed to get all applications", log.Any("error", err))
+			return nil, nil
+		}
+		for _, app := range apps.Applications {
+			applicationIDList = append(applicationIDList, app.ID)
+		}
+	} else {
+		applicationIDList = applicationIDs
+	}
+
+	for _, appID := range applicationIDList {
+		// Get the application
+		app, svcErr := es.applicationService.GetApplication(appID)
+		if svcErr != nil {
+			logger.Warn("Failed to get application for export",
+				log.String("appID", appID), log.String("error", svcErr.Error))
+			exportErrors = append(exportErrors, ExportError{
+				ResourceType: "application",
+				ResourceID:   appID,
+				Error:        svcErr.Error,
+				Code:         svcErr.Code,
+			})
+			continue // Skip applications that can't be found
+		}
+
+		// Convert to export format based on options
+		var content string
+		var fileName string
+
+		if options.Format == formatJSON {
+			// Convert to JSON format (could be implemented later)
+			logger.Warn("JSON format not yet implemented, falling back to YAML")
+			options.Format = formatYAML
+		}
+
+		templateContent, err := generateTemplateFromStructV2(app, app.Name)
```

Comment:
why this function is callad as v2 ?

Reply:
Fixed. Missed to remove the V2 as this was developed while the older logic was there.